packages/account-management/src/frontend/Registration/Registration.tsx

```diff
@@ -71,7 +71,13 @@ export const Registration: React.FC = () => {
 
   useEffect(() => {
     // check if password passes password policy
-    pass && !passwordPolicy.test(pass || '') ? setPassQualifies(false) : setPassQualifies(true);
+    if (pass && !passwordPolicy.test(pass || '')) {
+      setPassQualifies(false);
+      setCommonPassword(false);
+    } else {
+      setPassQualifies(true);
+      setCommonPassword(false);
+    }
```

Comment:
Can we simplify this?

```suggestion
    setPassQualifies(pass && passwordPolicy.test(pass))
    setCommonPassword(false)
```